src/super_gradients/training/transforms/processing.py

```diff
@@ -242,3 +287,20 @@ def default_ppyoloe_coco_processing_params() -> Tuple[List[str], Processing]:
     )
     class_names = COCO_DETECTION_CLASSES_LIST
     return class_names, image_processor
+
+
+def default_deciyolo_coco_processing_params() -> Tuple[List[str], Processing]:
+    """Processing parameters commonly used for training DeciYolo on COCO dataset."""
+    from super_gradients.training.datasets.datasets_conf import COCO_DETECTION_CLASSES_LIST
+
+    image_processor = ComposeProcessing(
+        [
+            ReverseImageChannels(),
+            DetectionLongestMaxSizeRescale(output_shape=(636, 636)),
+            DetectionCenterPadding(output_shape=(640, 640), pad_value=114),
+            StandardizeImage(max_value=255.0),
+            ImagePermute(permutation=(2, 0, 1)),
+        ]
+    )
+    class_names = COCO_DETECTION_CLASSES_LIST
+    return class_names, image_processor
```

Comment:
So this is actually what we had concerns over yesterday...
preprocessing parameters might vary. Class names might not even be relevant for future tasks for instance.
My suggestion - lets return a dictionary that can later on be passed to set_dataset_processing_params.
This way we can also pass the best iou and conf in terms of visuall sucess on COCO.

Reply:
We now return a dict d974313215196cc3c4210f1c49682040b93285cf